### PR TITLE
Update gulpfile.js to suport node files

### PR DESCRIPTION
**Pull request by barchero**
I've installed your yeoman generator but when include some library starts to crash because webpack can't access to the libraries of each library i was using.
i've solved this issue with this little modifications.
Enjoy it ;)

### CODE REVIEW

**Comment by jmeas** [comment]
Can you provide an example showing this issue, and explain a little more in-depth how these changes fix the problem? I've used this boilerplate quite a bit, and I've never had any issues importing from `node_modules`.

File: app/templates/gulpfile.js [hunk 1]
```diff
@@ -50,23 +51,38 @@ function lintGulpfile() {
...
       },
+      debug: true,
+      target: 'node',
       // Add your own externals here. For instance,
       // {
       //   jquery: true
       // }
       // would externalize the `jquery` module.
-      externals: {},
+      externals: nodeModules,
```
Comment (jmeas) [inline app/templates/gulpfile.js]: Is your goal to just externalize your node deps? If so, I'm not sure if that should be in the library out of the box. I'd be open to adding a wiki page on how to do it, though!

**Comment by barchero** [comment]
image

Here you have a little piece of the long error. That module is included from the library, not from my code.

**Comment by jmeas** [comment]
@barchero is there a project I can pull down to see the error?

**Comment by jmeas** [comment]
ping @barchero

**Comment by jmeas** [comment]
@barchero, I'm going to close this for now due to inactivity, but I'm very interested in hearing more about these changes! Just ping me here if you want to continue working toward getting this landed.

Or if anyone else sees this and thinks it should be landed, just let me know.

Thanks again for opening the PR, @barchero :v:

**Pull request closed without merging**